src/js/slide/slide.js

```diff
@@ -117,7 +116,7 @@ class Slide {
     this.updateContentSize();
     this.appendHeavy();
 
-    this.holderElement.appendChild(this.container);
```

Comment:
`holderElement` may be modified in process, use `this.holderElement`.

Reply:
fixed. But it's bad practice to mutate an object in process.